x-pack/plugins/lens/public/indexpattern_plugin/indexpattern.test.tsx

```diff
@@ -71,6 +72,11 @@ const expectedIndexPatterns = {
   },
 };
 
+const dragDropContext = {
+  dragging: undefined,
+  setDragging: jest.fn(),
```

Comment:
As written, this mock will track every call to `setDragging` across all these tests. Two solutions:

a) `jest.clearAllMocks();` in beforeEach
b) Don't use a constant `dragDropContext`, assign as needed